### DISCUSSION TITLE
Do I need additional thermostats if I have my own temperature sensor

### DISCUSSION THREAD

**Discussion by basvdl** [plugwise/python-plugwise#354] [Q&A]
Hi,

I have a Smile P1 and Anna thermostat (in the living room). I'm planning to buy a number of Tom thermostat knobs to implement zones heating. I have already a Aqara temperature sensor in every room

I was wondering can I get zoned heating working with only adding an Adam and Tom thermostats knobs and use my Aqara temperature sensor to tell admin via this integration what the current and preferred room temperature?

Can I get it to work without adding a thermostat to each and every room and basically have this set in HomeAssistant and update Admin using this integration?

Thanks!

**Comment by bouwew** [comment]
Hi @basvdl indeed you need to use an Adam to make this work.
Unfortunately, the Adam does not recognize the Aqara temperature sensor as a supported thermostat.
You will need to buy Lisa's or Jip's (which is an Aqara sensor with special Plugwise-firmware).

Please note: a Tom can also function as the room-thermostat. When there is only one Tom in a room and no Lisa or Jip present in the same room (actually I mean set up in the same room in the Plugwise App), then the temperature sensor of the Tom will be used in the regulation of the heating of the room.

**Comment by basvdl** [comment]
Hi @bouwew , Is there a way I can push my Aqara temp to the Adam or Tom every minute. I'm technically skilled enough to make it happen, the question is if Adam / Tom allows it.

Thinking about using my HA and node red for instance to update current room temperatures, and setting up schedules / activate heating.

Also avoiding emptying my bank account on additional Jip / Lisa. I can probably use the Tom thermostat and have some additional logic based on the actual room temperature to heat longer or shorter if the difference between Tom and actual room temperature is getting rather big.

**Comment by bouwew** [comment]
No, not that I know off. That would only work if the Adam would recognize it as a Jip.
I have tried to connect my Aqara temp sensor to the Adam, that works. But it's not recognized as a valid temperature sensor.

**Reply by basvdl** [reply]
To bad, I was hoping for similar feature as with tado described here: https://www.home-assistant.io/integrations/tado/#service-tadoset_climate_temperature_offset.

Thanks for you help!

**Comment by basvdl** [comment]
Sorry to bother you once more. I have a couple of question, which is not fully clear from the docs... Thanks in advance!

I'm thinking about adding an Adam HA, 9 x Tom and 2x Floor or Koen to my current setup (Anna + Smile). I would like to make sure that I can get the following out of the HomeAssistant (HA) integration.

1. Can I control the temperature or mode (away, night, home) for certain rooms using HA?
Use case: If no motion detected, HA will tell Adam to stop heating that room

2. Can I control the temperature or mode (away, night, home) for a specific Tom and/or Floor using HA?

3. Can I control the valve position (i.e. 0 to 100%) for a specific Koen using HA? I have feeling it is 0 or 100% based on the required plug...
Use case: We have 2 zones, bathroom and toilet, toilet is smaller so I would like to decrease the flow of water towards that zone.

4. Will Koen also send out a Heat demand request to Adam, when opening the valve or is this is based on a thermostat in the bathroom and/or toilet?

**Comment by bouwew** [comment]
1. Yes.
2. Yes if it the single device in a room as defined in the Plugwise App.
3. No, you will need to add a separate flow-restrictor if you want the have a flow difference. (As far as we know, there is no user-control of a Tom or Floor, can you please check if it can be done from the Plugwise App?)
4. It's based on a thermostat in the room.

About your question regarding changing the temperature offset, can you please check if it can be done from the Plugwise App?
If yes, please enter a Feature request under Issues.
